src/main/java/com/parsing/parsers/pdf/parsing/PageOCRPreparator.java

```diff
@@ -67,7 +67,6 @@ private List<Rectangle> extractTextFromRectangle(String filename, List<Rectangle
         itesseract.setDatapath(getTessDataPath());
         itesseract.setLanguage("ukr+eng");
         List<Rectangle> result = new ArrayList<>();
-        final Mat tableMat = Imgcodecs.imread(filename);
```

Comment:
why is it removed?

Reply:
don't using

Reply:
I'm not sure that if the result is not being used it means that it's pointless, ask Olexander about it please, he knows more about it.

Reply:
We talked that we can delete some unusual code, but it extends beyond this task.